R/deployAPI.R

```diff
@@ -0,0 +1,31 @@
+#' Deploy a Plumber API
+#'
+#' Deploys an application consisting of a series of plumber API routes.
+#' Only one of api or entry may be passed.
```

Comment:
The documentation seems to imply two parameters (`api` and `entry`) but only `api` appears in the function signature